lib/rules/utils/regular-expressions.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+const { RegExpValidator } = require("@eslint-community/regexpp");
+
+const REGEXPP_LATEST_ECMA_VERSION = 2022;
+
+module.exports.REGEXPP_LATEST_ECMA_VERSION = 2022;
```

Comment:
Can we avoid duplicating the constant value? We could write the module like this:

```js
"use strict";

const REGEXPP_LATEST_ECMA_VERSION = 2022;

function isValidWithUnicodeFlag(ecmaVersion, pattern) {
    // ...
}

module.exports = {
    REGEXPP_LATEST_ECMA_VERSION,
    isValidWithUnicodeFlag
};
```

Reply:
Also, can you please add `@fileoverview` and `@author` tags to the new file? Here's an example:

https://github.com/eslint/eslint/blob/1fbf1184fed57df02640aad4659afb54dc26a2e9/lib/rules/utils/ast-utils.js#L1-L4